misc/bazel/BUILD.toolchain.tpl

```diff
@@ -0,0 +1,12 @@
+load("@//misc/bazel:toolchain.bzl", "codeql_cli_toolchain")
```

Comment:
this file is missing.

Also, I think this entire toolchain stuff should be deleted until it's in a shape we can use it somewhere?

Reply:
ouch, I was supposed to have git stashed these veeery experimental toolchain stuff, sorry about this